### PR TITLE
E6S Raktapaksa ferostorm

### PR DESCRIPTION
**Pull request by mooondark**

### CODE REVIEW

File: ui/raidboss/data/05-shb/raid/e6s.js [hunk 1]
```diff
@@ -23,10 +23,15 @@
     },
     {
       id: 'E6S Ferostorm',
-      regex: Regexes.startsUsing({ source: 'Garuda', id: '4BF[EF]', capture: false }),
+      regex: Regexes.startsUsing({ source: ['Garuda', 'Raktapaksa'], id: ['4BF[EF]', '4C0[45]'], capture: false }),
+      regexDe: Regexes.startsUsing({ source: ['Garuda', 'Raktapaksa'], id: ['4BF[EF]', '4C0[45]'], capture: false }),
+      regexFr: Regexes.startsUsing({ source: ['Garuda', 'Raktapaksa'], id: ['4BF[EF]', '4C0[45]'], capture: false }),
+      regexJa: Regexes.startsUsing({ source: ['ガルーダ', 'ラクタパクシャ'], id: ['4BF[EF]', '4C0[45]'], capture: false }),
+      regexCn: Regexes.startsUsing({ source: '迦楼罗', id: '4BF[EF]', capture: false }),
```
Comment (quisquous) [inline ui/raidboss/data/05-shb/raid/e6s.js]: Can you update the Cn and Ko versions as well?

Comment (mooondark) [reply ui/raidboss/data/05-shb/raid/e6s.js]: I wish I can but I don't know what is the translation for Raktapaksa in these 2 languages...

File: ui/raidboss/data/05-shb/raid/e6s.js [hunk 2]
```diff
@@ -23,10 +23,15 @@
...
+      regexKo: Regexes.startsUsing({ source: '가루다', id: '4BF[EF]', capture: false }),
```
Comment (Una-P) [inline ui/raidboss/data/05-shb/raid/e6s.js]: Korea is still 5.0 so we don't know the official translation yet, literally translation is 락타팍사.
```suggestion
      regexKo: Regexes.startsUsing({ source: ['가루다', '락타팍사'], id: '4BF[EF]', capture: false }),
```

File: ui/raidboss/data/05-shb/raid/e6s.js (hunk 1 above)
Comment (quisquous) [reply ui/raidboss/data/05-shb/raid/e6s.js]: Could you remove these lines then? I think it's much more obvious that this needs a translation when there's no regex at all.  Having a regex there that's half done seems like something that could easily be missed in the future when the name is known.

**Pull request merged**